site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -173,4 +196,35 @@
         $("#peer-table").on("click", ".fa-pencil-alt", adminGradeablePeerForm);
         $('#admin-gradeable-peers-form').on("change", ":text", checkSubmitValid);
     });
-</script>
\ No newline at end of file
+    function downloadPeerDataAsCsv(table_id) {
+    // Select rows from table_id
+    var rows = document.querySelectorAll('table#' + table_id + ' tr');
+    // Construct csv
+    var csv = [];
+    header=['grader','student'];
+    csv.push(header.join(','));
+    for (var i = 1; i < rows.length; i++) {
```

Comment:
use `let  i = 1` instead of `var`, otherwise the variable i will actually be defined in the above scope and not be restricted to the for loop - this could cause variable pollution in other parts of the scope.

Reply:
done